runtime/xamarin-support.m

```diff
@@ -96,6 +101,20 @@
 	return result;
 }
 
+//
+// Returns the geopolitical region ID of the local timezone.
+// Previously we just provided the data to TimeZoneInfo.MonoTouch.cs and that
+// defaulted the ID to "Local", which is incorrect. 
+
+const char *
+xamarin_timezone_get_local_name ()
+{
+	NSTimeZone *tz = nil;
+	tz = [NSTimeZone localTimeZone];
+	NSString *name = [tz name];
+	return strdup ([name UTF8String]);
```

Comment:
This will crash if `name` is NULL (which should probably not happen, but I've seen strange data come out of these Apple API before, and they're a pain to debug if they only occur under certain circumstances).